projects/Mallard/src/screens/article-screen.tsx

```diff
@@ -1,29 +1,36 @@
 import React from 'react'
-import {
-    View,
-    Text,
-    StyleSheet,
-    ScrollView,
-    Button,
-    SafeAreaView,
-} from 'react-native'
-import { MonoTextBlock, HeadlineText } from '../components/styled-text'
+import { View, Text, StyleSheet, Button, Platform } from 'react-native'
+import { HeadlineText } from '../components/styled-text'
 import { useEndpoint } from '../hooks/use-fetch'
-import { Transition } from 'react-navigation-fluid-transitions'
 import { NavigationScreenProp } from 'react-navigation'
 import { color } from '../theme/color'
 import { metrics } from '../theme/spacing'
+import { SlideCard } from '../components/layout/slide-card'
+import {
+    useArticleAppearance,
+    WithArticleAppearance,
+    articleAppearances,
+} from '../theme/appearance'
 
+const notchInsetSize = Platform.OS === 'ios' ? 50 : 0
 const styles = StyleSheet.create({
     container: {
-        color: '#fff',
+        marginTop: notchInsetSize,
```

Comment:
Would using `<SafeAreaView>` mitigate this? Or does it do something undesirable?

Reply:
`SafeAreaView` accounts for scroll and makes animations weird. The design probably will require us to find out the value of the notch and do some math on it so i kept it as a constant for now with the idea of replacing it with the real thing, effectively giving us our own safeareaview